js/js.tests/test/org/jetbrains/kotlin/js/test/BasicWasmBoxTest.kt

```diff
@@ -48,8 +51,20 @@ abstract class BasicWasmBoxTest(
 
     private val spiderMonkey by lazy { SpiderMonkeyEngine() }
 
+    private val runIgnoredWasmTests =
+        getBoolean("kotlin.wasm.run.ignored.tests")
+
+    fun doTestWithCoroutinesPackageReplacement(filePath: String, coroutinesPackage: String) {
+        TODO("TestWithCoroutinesPackageReplacement are not supported")
+    }
+
     fun doTest(filePath: String) {
         val file = File(filePath)
+
+        val isIgnored = InTextDirectivesUtils.isIgnoredTarget(TargetBackend.WASM, file)
```

Comment:
Is it required for quicker tests or for something else?

Reply:
It used to be. Removed it.